Documentation/devel/release.md

```diff
@@ -19,8 +19,8 @@ Let's get started:
 
 The rkt version is [hardcoded in the repository](https://github.com/coreos/rkt/blob/master/configure.ac#L2), so the first thing to do is bump it:
 
-- Run `scripts/bump-release v0.16.0`.
-  This should generate two commits: a bump to the actual release (e.g. v0.16.0), and then a bump to the release+git (e.g. v0.16.0+git).
+- Run `scripts/bump-release v1.1.0`.
+  This should generate two commits: a bump to the actual release (e.g. v1.1.0), and then a bump to the release+git (e.g. v1.1.0+git).
```

Comment:
did you have to modify manually the second commits for scripts/acbuild-rkt-builder.sh and scripts/build-rir.sh?